lib/active_shipping/carriers/fedex.rb

```diff
@@ -662,10 +662,12 @@ def parse_tracking_response(response, options)
           description = event.at('EventDescription').text
           type_code = event.at('EventType').text
 
-          time          = Time.parse(event.at('Timestamp').text)
+          timestamp     = event.at('Timestamp').text
+          time          = Time.parse(timestamp)
+          zoneless      = (timestamp =~ /[-+]\d{2}:\d{2}$/).blank?
```

Comment:
I'm a fan of extracting regex into a named constant which explains what it does `TIMEZONE_FROM_DATE_REGEX`.